recaf-core/src/main/java/me/coley/recaf/code/InnerClassInfo.java

```diff
@@ -0,0 +1,37 @@
+package me.coley.recaf.code;
+
+public class InnerClassInfo implements ItemInfo {
+	private final String className;
+	private final String name;
+	private final String outerName;
+	private final String innerName;
+	private final int access;
+
+	public InnerClassInfo(String className, String name, String outerName, String innerName, int access) {
+		this.className = className;
+		this.name = name;
+		this.outerName = outerName;
+		this.innerName = innerName;
+		this.access = access;
+	}
+
+	public String getClassName() {
+		return className;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public String getOuterName() {
+		return outerName;
+	}
+
+	public String getInnerName() {
+		return innerName;
+	}
+
+	public int getAccess() {
+		return access;
+	}
+}
```

Comment:
Generally considering how this attribute can be confusing at first, adding some notes on _"this may seem odd but entries like `xyz` can exist when `<condition>`"_ when adding documentation later